packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -30,10 +30,8 @@ export const defaultLayouts = {
 	[ LAYOUT_TABLE ]: {
 		layout: {
 			primaryField: 'title',
+			mediaField: 'featured_media',
```

Comment:
List & Grid have "media fields", meaning there are special spots for rendering this field. What does it mean the "media field" of the table?